.changeset/moody-rice-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+"uploadthing": minor
```

Comment:
should probably be a patch since it is not adding any functionality, just fixing a missing type.